compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPULowerExecutableTarget.cpp

```diff
@@ -128,13 +128,13 @@ void LLVMCPULowerExecutableTargetPass::runOnOperation() {
   }
 
   ModuleOp moduleOp = variantOp.getInnerModule();
+  LLVMCPUPipelineOptions options;
```

Comment:
[nit] options -> pipelineOpts

“options” is just too generic IMHO (what happens if there’s some other set of “options”?)

Reply:
SGTM!